### ISSUE TITLE
HRPD_SETUP: Install and configure IBEX

### ISSUE THREAD

**Issue by kjwoodsISIS** [ISISComputingGroup/IBEX#5788]
As an `HRPD` scientist, I would like a setup PC, `HRPD_SETUP`, so that I can prepare experiments offline.

#### Acceptance Criteria
1. IBEX installed on `HRPD_SETUP`
1. IBEX configured on `HRPD_SETUP`
   * There will be no DAE on `HRPD_SETUP`
1. `HRPD` scripts migrated to `HRPD_SETUP`

#### Notes
1. `HRPD` scientists want to perform sample environment style experiments, such as calorimetry, using the same equipment as used on the beamline but without using the neutron beam - an 'offline' system would be useful.
1. `HRPD_SETUP` should be a clone of `HRPD` (except for the DAE).
2. Currently the Config is not a git repo it is just has some tables in it so the DAE could be run. A new config branch needs to be created and the current "HRPD" directory needs to be deleted. (It is not clear to me whether this needs to be a copy of HRPD config but with PVs pointing at HRPD_SETUP (or whatever the prefix is).

**Comment by rerpha** [comment]
Clearing the milestone for now as project board checks are complaining.

**Comment by Tom-Willemsen** [comment]
- IBEX is present on the cloned system, pointed it at NDXHRPD_SETUP
  * added to https://github.com/ISISComputingGroup/IBEX/wiki#instrument-information--hotfixes
- Have migrated configs & scripts over from HRPD as a starting point - hope is that scientists will be confident enough to edit these to match what HRPD_SETUP needs.
- NPort configured for HRPD's normal MOXA. I will explain the caveats around this to the scientists once the review is complete
  * I believe the kit is probably switched off during shutdown so nothing talking at present.

**Comment by JamesKingWork** [comment]
https://github.com/ISISComputingGroup/IBEX/pull/6195

**Comment by DominicOram** [comment]
Looks good, have created another ticket for adding to the inst list, which have put in as a flash review

**Comment by DominicOram** [comment]
Have we informed the scientists this is now done?

**Comment by Tom-Willemsen** [comment]
Will do that now that the review is complete.
